src/trajectory/Interpolated.cpp

```diff
@@ -100,6 +100,18 @@ void Interpolated::evaluateDerivative(double _t, int _derivative,
     _tangentVector.setZero();
   }
 
+  //if(_t==getStartTime())
+  if(_t==mWaypoints[0].t)
+  {
+      const auto segmentTime = mWaypoints[1].t - mWaypoints[0].t;
+      const auto alpha = (mWaypoints[1].t - mWaypoints[0].t) / segmentTime;
+
+      mInterpolator->getDerivative(mWaypoints[0].state, mWaypoints[1].state,
+              _derivative, alpha, _tangentVector);
+      _tangentVector /= segmentTime;
+      return;
```

Comment:
@mkoval I am taking _t == start_time as a specific case. So I will return here to skip running the other part of the function. This is an unfinished discussion. Do you still think we should support the derivative here?